docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,141 @@
+
+## Prerequisites
+
+**NOTE:** The public IPs and DNS records for Istio Ingress and remote environments gateway must exist before Kyma installation. Also, Kyma installer does not support clusters on AWS because AWS does not support assigning static IPs during creating ELB.
+
+Before you start the installation, check that you have access to a running cluster with the minimum Kubernetes version 1.10.
+
+To install Kyma, you need the following data:
+
+- IP address for Kyma Ingress
+- IP address for Remote Environments Ingress. For details, see the **Architecture** document for the Ingress-Gateway controller.
+- Domain name, for example `kyma.example.com`
+  - `gateway.kyma.example.com` points to Remote Environments Ingress IP address
+  - `*.kyma.example.com` points to Kyma Ingress IP address
+- Wildcard TLS certificate for your cluster domain that you can generate with **Let's Encrypt**
+- Certificate for Remote Environments. For details, see the **Overview** document for the Application Connector.
+
+Configure the Kubernetes API Server in the following manner:
+
+```
+"apiServerConfig": {
+    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota,PodPreset",
+    "--runtime-config": "batch/v2alpha1=true,settings.k8s.io/v1alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
+    "--cors-allowed-origins": ".*",
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--oidc-issuer-url": "https://dex.kyma.example.com",
+    "--oidc-client-id": "kyma-client",
+    "--oidc-username-claim": "email",
+    "--oidc-groups-claim": "groups"
+},
+"kubeletConfig": {
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--authentication-token-webhook": "true",
+    "--authorization-mode": "Webhook"
+}
+```
+
+## Installation
+
+1. Create the `kyma-installer` Namespace.
+
+Run the following command:
+
+```
+kubectl create ns kyma-installer
+```
+
+2. Fill in the `installer-config.yaml.tpl` template.
+
+The Kyma installation process requires installation data specified in the `installer-config.yaml` file. Copy the `installer-config.yaml.tpl` template, rename it to `installer-config.yaml`, and fill in these placeholder values:
+
+- `__TLS_CERT__` for the TLS certificate
+- `__TLS_KEY__` for the TLS certificate key
+- `__REMOTE_ENV_CA__` for the Remote Environments CA
+- `__REMOTE_ENV_CA_KEY__` for the Remote Environments CA key
+- `__EXTERNAL_IP_ADDRESS__` for the IP address for Kyma Ingress
+- `__DOMAIN__` for the domain name, for example `kyma.example.com`
+- `__REMOTE_ENV_IP__` for the IP address for Remote Environments Ingress
+- `__K8S_APISERVER_URL__` for the API server's URL
+- `__K8S_APISERVER_CA__` for your API Server CA
+- `__ADMIN_GROUP__` for the additional admin group. This value is optional.
+- `__ENABLE_ETCD_BACKUP_OPERATOR__` for enabling or disabling the `etcd` backup operator. Enter `true` or `false`.
```

Comment:
- `__ENABLE_ETCD_BACKUP_OPERATOR__` to enable or disable the `etcd` backup operator. Enter `true` or `false`.